content/docs/get-in-touch.md

```diff
@@ -19,7 +19,3 @@ Our GitHub org has many different repos, please make sure to pick the one approp
 * For general backend features or issues use the backend repo [github.com/jaegertracing/jaeger](https://github.com/jaegertracing/jaeger).
 * For frontend features or issues use [github.com/jaegertracing/jaeger-ui](https://github.com/jaegertracing/jaeger-ui).
 * For features or issues with client libraries, pick the [corresponding repository](/docs/client-libraries/#supported-libraries).
-
-## Report a security issue
```

Comment:
Could actually keep this section, with text "If you have a security issue to report, rather than a regular bug, please see..."

Reply:
I thought about that, but it would have been the third link to the page (the other two being the left-hand side menu and the footer):

![image](https://user-images.githubusercontent.com/13387/40126709-2e8554b8-592e-11e8-9fde-23deaac44340.png)